src/main/kotlin/racingcar/domain/car/CarAction.kt

```diff
@@ -0,0 +1,6 @@
+package racingcar.domain.car
+
+enum class CarAction(val distance: Int) {
+
+    MOVE(distance = 1), STOP(distance = 0);
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
    MOVE(1), STOP(0)
```

Reply:
https://github.com/next-step/kotlin-racingcar/pull/238/commits/709eb56a078cda4273ff7d661c9794ffc2812944 반영했습니다.